server/data/breachNoticeApiClient.ts

```diff
@@ -104,34 +112,25 @@ export interface SelectItem {
   selected: boolean
 }
 
-export interface RadioButton {
-  value: string
-  text: string
-  checked: boolean
+// Reference Data
+export interface ReferenceData {
+  code: string
+  description: string
 }
```

Comment:
I think this has moved too, we should update usages to the one in the other API client
```suggestion
```